### PR TITLE
Rework the SecurityPage for when there is no license present

### PR DESCRIPTION
**Pull request by simonychuang**
As described in https://github.com/Graylog2/graylog-plugin-enterprise/issues/3545, rework the page when there is no license present to make it look more like an information page rather than an error page.

/nocl

## How Has This Been Tested?

Tested manually on local environment

## Screenshots (if appropriate):
Screenshot 2023-04-20 at 2 11 47 PM

Screenshot 2023-04-20 at 2 12 04 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by simonychuang** [comment]
I was having trouble creating a custom component using the `<Jumbotron>`, it was not taking the `theme` value properly from the `ThemeModeContext`. I got it working by just putting the image there from the mockups and wrapping it in a `<a>` to have it redirect to the appropriate url. If anyone has any idea how to make it work with the `<Jumbotron>` component I am open to suggestions.

**Review by zeeklop — APPROVED** [review]
Looks way better, good work Simon!

When I clicked on the link to learn more about security, the docs asked me for a password. Is that ok?

image

**Comment by rich-graylog** [comment]
@zeeklop Just double checked on that page, it will be corrected before release, so we're good to go there 👍

**Comment by kingzacko1** [comment]
Looks awesome, @simonychuang! Nice work!

File: changelog/unreleased/issue-3545.toml
Comment (kingzacko1) [inline changelog/unreleased/issue-3545.toml:1]: I think we can probably do without a changelog file for this PR since it is purely cosmetic.

**Review by kingzacko1 — APPROVED** [review]
Looks solid and adapts to changing light/dark theme well. Outside of removing the changelog entry, looks ready for merge.

**Pull request merged**